Hotel/src/hotel/Terminal.java

```diff
@@ -0,0 +1,67 @@
+package hotel;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class Terminal {
+    public void startTerminal(String previousInput, String currentScreen){
+        Scanner terminalInput = new Scanner(System.in);
+        System.out.println(currentScreen);
+        String newInput = terminalInput.nextLine();
+        String input = previousInput != null? previousInput +","+ newInput:  newInput;
```

Comment:
I find these kind of if statements hard to read, especially for newer programmers, we should take them into account.